metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -546,6 +546,7 @@ def tile_custom_properties(tile: powerbi_data_classes.Tile) -> dict:
             entity_urn=chart_urn,
             aspect_name=Constant.CHART_INFO,
             aspect=chart_info_instance,
+            change_type=ChangeTypeClass.PATCH,
```

Comment:
patches have a different format than normal aspects, so this doesn't really work

also, I would like to implement this as a source_helper if possible. Would be similar to (or ideally, part of) the `auto_incremental_lineage` helper

Note that the existing `auto_incremental_lineage` needs to change as well, since we now have support for patching CLL on the backend